ethereum/rpc/backend/backend.go

```diff
@@ -241,7 +241,7 @@ func (e *EVMBackend) HeaderByNumber(blockNum types.BlockNumber) (*ethtypes.Heade
 	switch blockNum {
 	case types.EthLatestBlockNumber:
 		if currentBlockNumber > 0 {
-			height = int64(currentBlockNumber - 1)
+			height = int64(currentBlockNumber)
 		}
 	case types.EthPendingBlockNumber:
 		if currentBlockNumber > 0 {
```

Comment:
for `pendingBlockNumber`, not sure which value we should return

Reply:
I think there might have to be a global solution for this.. as there are different endpoints that face a similar issue due to the same problem [more info here](https://github.com/tharsis/ethermint/issues/215)

Reply:
I think we don't have the concept of pending block here, so it should just be the same as latest one.

Reply:
yeah there is no pending block, just "pending" txs from the mempool